internal/config/config_test.go

```diff
@@ -675,6 +675,9 @@ func TestLoad(t *testing.T) {
 				cfg.Storage = StorageConfig{
 					Type: GitStorageType,
 					Git: &Git{
+						Storage: GitStorageConfig{
```

Comment:
nit: I think its a bit repetitive for the config to be `storage.git.storage`, is there another word we could use ? maybe `backend` ? would go well with other config options like `cache.backend` that we already have, where `memory` is an option

wdyt?

Reply:
Yeah I know what you mean. Backend works for me.